src/Traits/LogsActivity.php

```diff
@@ -61,11 +68,17 @@ protected static function eventsToBeRecorded(): Collection
             return collect(static::$recordEvents);
         }
 
-        return collect([
+        $events = [
             'created',
             'updated',
             'deleted',
-        ]);
+        ];
+
+        if(in_array('Illuminate\Database\Eloquent\SoftDeletes', class_uses(__CLASS__))) {
```

Comment:
`class_uses()` returns an array - i can't use collection methods on that.
http://php.net/manual/de/function.class-uses.php

i could make a collection first if you prefer:
`if(collect(class_uses(__CLASS__))->contains(\Illuminate\Database\Eloquent\SoftDeletes::class))`